test/doc/Main.hs

```diff
@@ -16,6 +16,7 @@ main =
     , "src/PostgREST/Query/SqlFragment.hs"
     , "src/PostgREST/ApiRequest/Preferences.hs"
     , "src/PostgREST/ApiRequest/QueryParams.hs"
+    , "src/PostgREST/Response/Performance.hs"
```

Comment:
Oh, so doctests weren't running for this module?

Reply:
Yeah, they weren't.